test/custom_operator/test_type_convertion.py

```diff
@@ -0,0 +1,47 @@
+# Owner(s): ["module: pt2-dispatcher"]
+from __future__ import annotations
+
+from typing import List, Optional, Sequence, Tuple, Union  # noqa: F401
```

Comment:
Why can't this file just go into test/test_custom_ops.py? That's where we currently have all of the infer_schema tests

Reply:
moved